webui/react/src/styles/icons.scss

```diff
@@ -74,6 +74,7 @@
 .icon-star::before { content: '\e90a'; }
 .icon-tasks::before { content: '\e920'; }
 .icon-tensor-board::before { content: '\e90b'; }
+.icon-tensorboard::before { content: '\e90b'; } // one word? matches the api response
```

Comment:
I can't remember exactly where but there's an area that takes camel case and converts to skewer case for icons. And when we did a renaming of `Tensorboard` to `TensorBoard` this change made it easier without needing a separate icon map.

Reply:
do you think it'd would be fine to have definitions for both in the name of ease of use?

Reply:
I'd say just keep the kebob one and drop the one word and whoever is making use of it or the reviewer, one of them will catch it when they render the view using the icon.